code/game/world.dm

```diff
@@ -35,6 +36,8 @@ GLOBAL_LIST_INIT(map_transition_config, MAP_TRANSITION_CONFIG)
 	populate_robolimb_list()
 
 	Master.Initialize(10, FALSE)
+	TgsInitializationComplete()
```

Comment:
TG handles this in MC; any reason it's different here?

Reply:
You definitely want this AFTER initializations. It tells TGS that the server is done loading. Nothing is done with the information yet, but it may be in the future.

Reply:
To be clear, this runs immediately

Reply:
So this is why my DMAPI validated instantly